features/step_definitions/collections_steps.rb

```diff
@@ -21,3 +21,31 @@
     end
   end
 end
+
+Then(/^I am in the "(.*?)" variant of the education navigation test$/) do |variant|
+  ab_cookie = page.driver.cookies["ABTest-EducationNavigation"]
+
+  assert_equal(
+    variant,
+    ab_cookie.value,
+    "We have been assigned to the incorrect variant"
+  )
+
+  assert(
+    ab_cookie.expires,
+    "Expected the cookie to have an expiry date"
+  )
+end
+
+Then(/^I stay on bucket "(.*?)" when I keep visiting "(.*?)"$/) do |variant, path|
+  20.times do
+    response = visit path
+    education_ab_cookie = page.driver.cookies['ABTest-EducationNavigation']
```

Comment:
Really minor point: could the step name mention education navigation to make it clearer that the step checks for this specific cookie?